### PR TITLE
chore(PPDSC-2644): update current svgs and loader

### PR DESCRIPTION
**Pull request by mutebg**
PPDSC-2644

**What**

1. Background - svg exporter shoud add id to every single svg so that can be loaded via use tag
2. What did you do - updated all svgs to have default id, changed the loader component, updated the exported to add id to svg
3. What does the reviewers should expect

URL witch changes WITHOUT svg files -> https://github.com/newscorp-ghfb/newskit/pull/522/files?file-filters%5B%5D=.ts&file-filters%5B%5D=.tsx&show-viewed-files=true

**I have done:**
- [ ] Written unit tests against changes
- [ ] Written functional tests against the component and/or NewsKit site
- [ ] Updated relevant documentation

**I have tested manually:**
- [ ] The feature's functionality is working as expected on Chrome, Firefox, Safari and Edge
- [ ] The screen reader reads and flows through the elements as expected.
- [ ] There are no new errors in the browser console coming from this PR.
- [ ] When visual test is not added, it renders correctly on different browsers and mobile viewports (Safari, Firefox, small mobile viewport, tablet)
- [ ] The Playground feature is working as expected

**Before:**

**After:**

**Who should review this PR:**

**How to test:**

### CODE REVIEW

**Comment by newskit-engineering** [comment]
You can preview these changes on:
  Documentation Site: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2644-svg-exported-id
  Storybook: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2644-svg-exported-id/storybook

**Review by JohnTParsons — APPROVED** [review]

**Review by baburay23 — DISMISSED** [review]

**Comment by Xin00163** [comment]
981 😲

**Review by Vanals — DISMISSED** [review]

File: site/components/svg-previewer/__tests__/__snapshots__/svg-previewer.test.tsx.snap [hunk 1]
```diff
@@ -14,8 +14,8 @@ exports[`SvgPreviewer should change theme colors correctly when selecting a diff
   display: grid;
   grid-template-columns: 1fr 1fr;
   -webkit-box-pack: justify;
-  -webkit-justify-content: sapce-between;
```
Comment (Vanals) [inline site/components/svg-previewer/__tests__/__snapshots__/svg-previewer.test.tsx.snap:17]: @mutebg  should we not also have the new `id` appearing in the snapshots 🤔 ?

Comment (Vanals) [reply site/components/svg-previewer/__tests__/__snapshots__/svg-previewer.test.tsx.snap:17]: If only exist in the file we download I thought I added a test to snap the content of the file, maybe a false memory 🤔 .

Comment (mutebg) [reply site/components/svg-previewer/__tests__/__snapshots__/svg-previewer.test.tsx.snap:17]: that's `site/components/svg-previewer/__tests__/expected-file-mock.tsx` isnt't it?
btw I will need your help to test it in figma, not sure how to do that

**Review by Vanals — APPROVED** [review]

**Review by baburay23 — APPROVED** [review]

**Pull request merged**